Common subdirectories: dvbinspector-release_1_19_2-orig/.github and dvbinspector-release_1_19_2-patched/.github
diff -u dvbinspector-release_1_19_2-orig/pom.xml dvbinspector-release_1_19_2-patched/pom.xml
--- dvbinspector-release_1_19_2-orig/pom.xml	2024-08-04 16:01:41.000000000 -0400
+++ dvbinspector-release_1_19_2-patched/pom.xml	2024-08-17 23:56:27.166902888 -0400
@@ -6,7 +6,7 @@
 	<groupId>nl.digitalekabeltelevisie</groupId>
 	<artifactId>DVBinspector</artifactId>
 	<packaging>jar</packaging>
-	<version>1.19.1</version>
+	<version>1.19.2</version>
 	<description>DVB Inspector is an open-source DVB analyzer, written in java</description>
 	<name>DVBinspector</name>
 
Common subdirectories: dvbinspector-release_1_19_2-orig/.settings and dvbinspector-release_1_19_2-patched/.settings
Common subdirectories: dvbinspector-release_1_19_2-orig/src and dvbinspector-release_1_19_2-patched/src
